app/src/main/java/org/alphatilesapps/alphatiles/Start.java

```diff
@@ -1548,6 +1608,235 @@ public ArrayList<String> parseWordIntoTiles(String parseMe) {
             return parsedWordArrayTemp;
         }
 
+        /*
+        LM: parseWordIntoTiles does not currently take into consideration symbols of type X because we don't have
+        data to show how these tiles pattern in Thai script language packs yet. 03-2023
+         */
+        public ArrayList<String> parseWordIntoTiles(String stringToParse, String wordInLOP) {
```

Comment:
Would you mind explaining what the stringToParse parameter is for? At least based on command F it seems like the method is always called with the same value passed in for the two parameters, with stringToParse only being used once. Thanks!

Reply:
I just added a new comment to the method header about those parameters. Let me know if you have any further questions. 
TLDR: Sometimes the stringToParse isn't something in the wordlist, but rather a pseudoword for an incorrect answer choice. But we need real word information to know the correct way to parse Thai/Lao tiles.